sagemaker-pipelines/tabular/tensorflow2-california-housing-sagemaker-pipelines-deploy-endpoint/tensorflow2-california-housing-sagemaker-pipelines-deploy-endpoint.ipynb

```diff
@@ -0,0 +1,1244 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #32.        # TODO - ADD YOUR CODE TO SEND EMAIL...
</pre><p>Can this code be provided instead of a todo item? Also, that linked doc lists a lot of prerequisites - this seems like it is getting out of scope with a lot of additional manual work obfuscated away. If I were a customer looking for a code example for sending an email, I would be annoyed. At least state earlier in the notebook that the email-sending is left as an exercise or for the customer to implement themselves?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/2865/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='20'/>

Reply:
I omitted the reference to the docs and added a note for the reader to implement the code by the framework they choose